### PR TITLE
[unicode.org] Add ruleset.

### PR DESCRIPTION
**Pull request by katrinleinweber**

### CODE REVIEW

**Comment by J0WI** [comment]
Finally they support HTTP,S, thanks for adding this our rulesets!
Please have a look at those two sections in our docs:
https://github.com/EFForg/https-everywhere/blob/master/CONTRIBUTING.md#enumerating-subdomains
https://github.com/EFForg/https-everywhere/blob/master/CONTRIBUTING.md#non-working-hosts

**Comment by J0WI** [comment]
Please have a look at Travis. Our tests exposed some errors.
We use to put the comments on the very top of the ruleset.

**Comment by katrinleinweber** [comment]
Sorry for the above mistakes! I hope this is correct now :-)

File: src/chrome/content/rules/unicode.org.xml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<ruleset name="unicode.org">
+
+	<target host="unicode.org" />
+	<target host="www.unicode.org" />
+	<target host="cldr-smoke.unicode.org" />
+	<target host="st.unicode.org" />
+	
+<!-- Non-working hosts
```
Comment (J0WI) [inline src/chrome/content/rules/unicode.org.xml]: Please move the whole comment above the ruleset.

File: src/chrome/content/rules/unicode.org.xml [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+        Redirect to HTTP:
+        	cldr-build.unicode.org
+        	
+        "Secure Connection Failed"
+       		cldr.unicode.org
```
Comment (J0WI) [inline src/chrome/content/rules/unicode.org.xml]: Please use a consistent code style (only tabs to indent lines, colon after error type, no quotes)

File: src/chrome/content/rules/unicode.org.xml [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+        	blog.unicode.org
+        	uli.unicode.org
+        	
+        SSL_ERROR_BAD_CERT_DOMAIN
```
Comment (J0WI) [inline src/chrome/content/rules/unicode.org.xml]: Just use the wording "Mismatch"

**Comment by katrinleinweber** [comment]
@J0WI Thanks for the hints and my apologies for the messiness! Can you recommend an editor or IDE that you consider well-suited for XML? My Sublime Text v3 didn't highlight the tab/spaces problem.

**Comment by J0WI** [comment]
Thanks, LGTM!

We have an [EditorConfig](https://github.com/EFForg/https-everywhere/blob/master/.editorconfig) which is supported by some Plugins for Sublime Text.

**Pull request merged**